web/src/lib/components/nostr/Button.svelte

```diff
@@ -29,37 +29,91 @@
 
         message += '\n' + pmURL;
 
-        pool.connectAndSendMessage({
-            message,
-            nostrRoomId,
-        });
+        document.getElementById('nostrTextToSend').value = message;
```

Comment:
Using `getElementById` works, but is a bit ugly to do in svelte. A nicer way would be declaring `message` outside the function and using `bind:value={message}` and `bind:checked={textConfirmationVisible}`.

Reply:
Of course. I don't know what I was thinking... Thanks!